src/components/loan-investment-calculator/summary.js

```diff
@@ -68,25 +78,8 @@ const Summary = props => {
           />
         </Row>
       </div>
-      <div>
-        <h4>How it works</h4>
-        <p>
-          <b>
-            You make {formatMoney(props.optCost)} more by choosing the{' '}
-            {props.bestOption.term} year mortgage
-          </b>
-          . <b>{formatMoney(props.optCost)}</b> is the difference between your{' '}
-          {props.shorterOption.term} year investment total (
-          <b>{formatMoney(props.shorterOption.fv)}</b>) and your{' '}
-          {props.longerOption.term} year investment total (
-          <b>{formatMoney(props.longerOption.fv)}</b>).
-        </p>
-      </div>
-      <Row noGutters>
-        <ScenarioCol
-          option={props.shorterOption}
-          isBestOption={props.shorterOption === props.bestOption}
-        >
+      <Row noGutters className='mt-3'>
```

Comment:
sug: Rather than noGutters, should both BoxCol and ScenarioCol have less padding?

Reply:
Not sure, I'm still getting ahold of these. What I noticed is that the boxes weren't really aligned before nogutters and that the upper one was escaping the app's container for some reason. Adding `noGutters` solved that issue.

![image](https://user-images.githubusercontent.com/9560798/65165503-cd877e00-d9f3-11e9-823c-e762d449da6b.png)

Reply:
@hdavidzhu I actually am not sure what to do here. If there's a way to achieve the same result without using `noGutters` and other classes please be explicit about it since I am still trying to get familiar with these classes

Reply:
🤷‍♂ I won't get to it until after work, so feel free to merge before then. Perhaps I'll figure something out if I come across it again!